kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -2933,6 +2950,65 @@ def ceph_host_heartbeat_check(self, req):
+            "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "
+            "-o ConnectTimeout=5 -o ServerAliveInterval=5 -o ServerAliveCountMax=2 "
+            "-o BatchMode=no %s@%s %s"
+        ) % (ssh_timeout, private_key_file, target_port, target_user, target_ip,
+             linux.shellquote(remote_cmd))
+
+        logger.info("fence vm[%s] on suspect host[%s] via this peer" % (vm_uuid, target_ip))
+        try:
+            try:
+                s = shell.ShellCmd(ssh_argv)
+                s(False)
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)